### PR TITLE
Allow Google Drive module to upload to a specified folder

### LINKED ISSUE #14: Change default google drive permissions
Change permissions so that GitHub users do not have to ask permission to view uploaded files.

### PR DESCRIPTION
**Pull request by tcbyrd**
Closes #14 

This adds the `parents` key to the resource object to allow the uploaded PDF to be stored in a folder that is automatically shared with everyone.

### CODE REVIEW

File: gdrive/src/uploadPdf.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ function uploadPDF(auth, file) {
   service.files.create({
     resource: {
       name: file.name,
-      mimeType: 'application/pdf'
+      mimeType: 'application/pdf',
+      parents: ['0Bx7UhUK81Pz1SE9YZ1JUR0FPbmc']
```
Comment (azizshamim) [inline gdrive/src/uploadPdf.js]: Is this the specific ID of the folder? Can this be configurable?

**Review by azizshamim — CHANGES_REQUESTED** [review]

File: gdrive/src/uploadPdf.js (hunk 1 above)
Comment (tcbyrd) [reply gdrive/src/uploadPdf.js]: Yeh, this is a specific folder in our Google Apps account. Could definitely be added to the [`google-config.json`](https://github.com/bryancross/key2pdf/blob/refactoring-to-modules/config/google-config-example.json).

**Comment by tcbyrd** [comment]
The way the Google Drive module is currently written, it gets its config from a subfolder but the rest of the config is loaded from a folder in the root directory (`key2pdf/gdrive/config` vs `key2pdf/config`). Not sure what the best practice is here. There are only 2 variables in `google-config.json`, so they could easily be added to `job-template.json`. Thoughts?

**Review by bryancross — APPROVED** [review]
We'll merge the config stuff in a later PR :)  These changes make this more than an MVP.  It's AWESOME!

**Comment by bryancross** [comment]
@tcbyrd I was looking at that as well when I reviewed your original code.  I think it's fine to merge this PR, and we can tidy up the config stuff later.  

BTW, this is 💯 work!  Super :cool:

**Pull request merged**